stubs/stripe/@tests/stubtest_allowlist.txt

```diff
@@ -1,5 +1,6 @@
 # The following methods have custom classmethod decorators
 stripe\..*\.delete
 stripe\..*PaymentIntent\.confirm
-stripe\.api_resources\..*\.SearchableAPIResource\.search  # Not defined on the actual class in v3, but expected to exist. 
-stripe\.api_resources\..*\.SearchableAPIResource\.search_auto_paging_iter  # Not defined on the actual class in v3, but expected to exist. 
+stripe\..*TestClock\.advance
```

Comment:
Do we need the wildcard? Seems like `stripe.api_resources.test_helpers.test_clock.TestClock.advance` would work.